kubernetes/ok-staging-worker.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: ok-worker
+spec:
+  replicas: 5
```

Comment:
5 is a bit high considering we won't have a lot of demand - but it should be fine.